TADashboard/src/main/java/page_objects/GeneralSettingsPage.java

```diff
@@ -0,0 +1,18 @@
+package page_objects;
+
+import com.logigear.control.common.imp.Button;
+import com.logigear.control.common.imp.Link;
+import com.logigear.driver.DriverUtils;
+
+public class GeneralSettingsPage {
```

Comment:
can you show me GeneralSetting page in the UI?

Reply:
![image](https://user-images.githubusercontent.com/34290752/133398151-9b0adb89-b45c-448b-8826-7205a13d3057.png)

Reply:
this is DataProfilesGeneralSettingPage